_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -220,169 +233,224 @@ In this curriculum, we will go through all the background topics necessary to un
      
      One of the roots of this polynomial satisfies \(|z|>1\) and this implies that the method is not zero-stable.
      </p>
+     </details>
+     <details><summary>Solution to Exercise 12.19</summary>
      <p>
-     Exercise 12.19:
      The first equation can be found by substituting \(f(t,y) = \lambda y\) in equation (12.51) in the book and by solving for \(k_1,k_2\) (it is a \(2\times 2\) linear system). Substituting the values of \(A\) and \(b\) from the Butcher tableau in this formula and in the one right before equation (12.51) in the book, and simplifying, we get the formula for \(R(\lambda h)\). Finally, \(p\) and \(q\) are given by \(p,q=-3\pm i \sqrt{3}\). One can see that this implies \(|R(z)|<1\) if \(Re(z) <0\) and thus the method is A-stable.
      </p>
      </details>
 
 <br />
 
 # 3 ResNets
-  **Motivation**: The introduction of Residual Networks (ResNets) made possible to train very deep networks. In this section we study some residual architectures variants and their properties. We then look into how ResNets approximates ODEs and how this interpretation can motivate neural net architectures and new training approaches.  This is important in order to understand the basic models underlying Neural ODEs and gain some insights on their connection to numerical solutions of ODEs.
+  **Motivation**: The introduction of Residual Networks (ResNets) made it possible to train very deep networks. In this section, we study residual architectures and their properties. We then look into how ResNets approximates ODEs and how this interpretation can motivate neural net architectures and new training approaches.  This is important in order to understand the basic models underlying Neural ODEs and gain some insights into their connection to numerical solutions of ODEs.
 
   **Topics**:
 
-  1. ResNets
-  2. ResNets and ODEs
+  1. ResNets.
+  2. ResNets and ODEs.
+
+  **Notes**: In this [class](/assets/nodes_notes/week3.pdf), we defined and briefly discussed residual network architecture. We then looked at a stability notion for ResNets, derived from the connection with ODEs discretisation, and to a simple way to make such architectures reversible.
```

Comment:
"We then looked at a stability notion for ResNets, derived from the connection with ODEs discretisation, and to a simple way to make such architectures reversible." --> "We then looked at a stability notion for ResNets that is derived from the connection with the discretization of ODEs. From those, we considered a simple way to make such architectures reversible."

"connection with ODEs discretisation" --> "connection with the discretization of ODEs"

Reply:
fixed